app/models/concerns/orchestration/tftp.rb

```diff
@@ -30,6 +30,10 @@ def tftp
     subnet.tftp_proxy if tftp?
   end
 
+  def tftp_smart_proxy
+    subnet.tftp
+  end
+
```

Comment:
We use `subnet.proxy_name` alot across our codebase, is this an attempt to somehow consolidate this? Frankly, I'd prefer keeping this as is or renaming the `tftp` method or introducing helper in the subnet actually.